parm/config/config.wave

```diff
@@ -56,6 +62,10 @@ export FHOUT_WAV=${FHOUT_WAV:-3}
 export FHMAX_HF_WAV=${FHMAX_HF_WAV:-120}
 export FHOUT_HF_WAV=${FHOUT_HF_WAV:-1}
 
+# Output stride
+export WAV_WND_HOUR_INC=1 # This value should match with the one used in
```

Comment:
? Is this from f/c-c?

Reply:
Yes, it was in the example configs I was given, see: /work/noaa/global/wkolczyn/save/global-workflow/expdir/example/config.wave

Let me know if we don't want this, thanks!

Reply:
Why don't we just keep this variable in config.waveprep like it is in develop and remove from config.wave.  I don't remember why f/c-c moved this, but keeping it in waveprep and not doubly defining it is probably the way to go.

Reply:
Will do, thanks!